src/snapshot/tosvg.js

```diff
@@ -118,6 +118,20 @@ module.exports = function toSVG(gd, format, scale) {
             if(fv && fv === 'normal') {
                 txt.style('font-variant', undefined);
             }
+
+            // Drop none text-shadow, text-transform and text-decoration-line to reduce the size
+            var ts = this.style.textShadow;
+            if(ts && ts === 'none') {
+                txt.style('text-shadow', undefined);
+            }
+            var tt = this.style.textTransform;
+            if(tt && tt === 'none') {
+                txt.style('text-transform', undefined);
+            }
+            var td = this.style.textDecorationLine;
+            if(td && td === 'none') {
+                txt.style('text-decoration-line', undefined);
```

Comment:
Should we be doing this even in interactive mode? ie whenever we would be setting one of these attrs to `'none'` we instead set it to `undefined`?

Reply:
Addressed in [9416441](https://github.com/plotly/plotly.js/pull/6983/commits/94164413ec95370ef18eff39b083884019c3e55e).